dist/constants.json

```diff
@@ -8919,1235 +8919,1235 @@
-        "label": "NEWPORT VA CLINIC"
+        "label": "Newport VA Clinic"
       },
       {
         "value": "531GH",
-        "label": "BURNS VA CLINIC"
+        "label": "Eastern Oregon VA Clinic"
       },
       {
         "value": "648GI",
-        "label": "PORTLAND VA CLINIC"
+        "label": "Portland VA Clinic"
       },
       {
         "value": "648GJ",
-        "label": "LOREN R KAUFMAN VA CLINIC"
+        "label": "Loren R. Kaufman VA Clinic"
       },
       {
         "value": "653QA",
-        "label": "DOWNTOWN EUGENE VA CLINIC"
+        "label": "Downtown Eugene VA Clinic"
       },
       {
         "value": "648",
-        "label": "PORTLAND VA MEDICAL CENTER"
+        "label": "Portland VA Medical Center"
       },
       {
         "value": "653",
-        "label": "ROSEBURG VA MEDICAL CENTER"
+        "label": "Roseburg VA Medical Center"
       },
       {
         "value": "692",
-        "label": "WHITE CITY VA MEDICAL CENTER"
+        "label": "White City VA Medical Center"
       }
     ],
     "OK": [
       {
         "value": "623BY",
-        "label": "TULSA, OK CBOC"
+        "label": "Ernest Childers Department of Veterans Affairs Outpatient Clinic"
       },
       {
         "value": "623GA",
-        "label": "MCALESTER VA CLINIC"
+        "label": "McAlester VA Clinic"
       },
       {
         "value": "635GA",
-        "label": "LAWTON/FT. SILL VA OUTPATIENT CLINIC"
+        "label": "Lawton VA Clinic"
       },
       {
         "value": "635GC",
-        "label": "BLACKWELL CBOC"
+        "label": "Blackwell VA Clinic"
       },
       {
         "value": "635GD",
-        "label": "ADA CBOC"
+        "label": "Ada VA Clinic"
       },
       {
         "value": "635HB",
-        "label": "ARDMORE VETERANS CENTER CLINIC"
+        "label": "Ardmore VA Clinic"
       },
       {
         "value": "564GE",
-        "label": "JAY CBOC"
+        "label": "Jay VA Clinic"
       },
       {
         "value": "623GB",
-        "label": "VINITA CBOC"
+        "label": "Vinita VA Clinic"
       },
       {
         "value": "635GE",
-        "label": "STILLWATER CBOC"
+        "label": "Stillwater VA Clinic"
       },
       {
         "value": "635GF",
-        "label": "ALTUS CBOC"
+        "label": "Altus VA Clinic"
       },
       {
         "value": "635GG",
-        "label": "ENID CBOC"
+        "label": "Enid VA Clinic"
       },
       {
         "value": "635QB",
-        "label": "SOUTH OKLAHOMA CITY VA CLINIC"
+        "label": "South Oklahoma City VA Clinic"
       },
       {
         "value": "623QC",
-        "label": "YALE AVENUE VA CLINIC"
+        "label": "Yale Avenue VA Clinic"
       },
       {
         "value": "635QC",
-        "label": "FOURTEENTH STREET VA CLINIC"
+        "label": "Fourteenth Street VA Clinic"
       },
       {
         "value": "623",
-        "label": "MUSKOGEE, OK VAMC"
+        "label": "Jack C. Montgomery Department of Veterans Affairs Medical Center"
```

Comment:
- Consider abbreviated label for "Department of Veterans Affairs"
- Let's confirm this name change.

Reply:
City is MUSKOGEE, OK. So, this LGTM.